### PR TITLE
[Ready] Surface popular topics (flagged)

### PR DESCRIPTION
**Pull request by charypar**
This is for interested people to see what the popular topics API is returning and give hints at what it should be returning. The final design treatment will probably be different

# to do
- [x] fetch topics from Popular API
- [x] allow limiting the amount of topics
- [x] surface in the page
- [x] flag

### CODE REVIEW

File: server/graphql/backend-adapters/popular-api.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class PopularAPI {
+	constructor(cache) {
+		this.cache = cache;
+	}
+
+	topics(ttl = 50) {
+		const url = 'https://ft-next-popular-api.herokuapp.com/topics?apiKey=' + process.env.POPULAR_API_KEY;
+
+		return this.cache.cached(`popular-api.topics`, ttl, () => {
```
Comment (ironsidevsquincy) [inline server/graphql/backend-adapters/popular-api.js]: no need for backticks

File: views/partials/lead-today.html [hunk 2]
```diff
@@ -1,34 +1,46 @@
 {{#if items}}
 	<section class="flexipod flexipod--lead-today" id="site-content" data-trackable="lead-today" role="region" tabindex="0" aria-labelledby="lead-pod-title">
-		<div class="article-group" role="presentation">
-			{{#slice leads limit=1}}
-				<div class="article no-ff-padding">
-					{{> story-card related=true showSummary=true }}
-				</div>
+		{{#if flags.popularTopics}}
+		<div class="pod-heading" role="presentation">
```
Comment (ironsidevsquincy) [inline views/partials/lead-today.html]: can put a `data-trackable="popular-topics"` or something on he, so we can group clicks links within it

**Comment by ironsidevsquincy** [comment]
:+1: good stuff, screenshot?

**Comment by charypar** [comment]
topics

**Comment by charypar** [comment]
Needs better design treatment for public release, but it'll do for now until we're happy with what it gives us.

File: views/partials/lead-today.html (hunk 2 above)
Comment (charypar) [inline views/partials/lead-today.html]: It's on the `<ul>` below

**Pull request merged**